src/Data/DateFormat/DateFormat.php

```diff
@@ -78,4 +92,14 @@ public function toString() : string
     {
         return implode('', $this->format);
     }
+
+    public function __toString() : string
+    {
+        return $this->toString();
+    }
+
+    public function format(\DateTimeImmutable $datetime) : string
```

Comment:
This would be $format->format(), and is sort of the opposite direction of php native function.
Maybe we could "apply" it to a $datetime? applyTo?

Reply:
Love it =)

Reply:
Done...